amaskiellEG/slick
# Feature | SlickJS for cash

# Description

Replaces portions of SlickJS reliant on jQuery so that the library can be utilized with [`cash`](https://github.com/fabiospampinato/cash).

# Relevant Tickets:

_TODO_ 🙃 

# How To Test

There are now 2 HTML playground files: `indexWithjQuery.html` and `indexWithCash.html`
- an easy way to open them from terminal while in this repo's top-level directory would be to run `open indexWithjQuery.html` or `open indexWithCash.html`
- there maybe be some slightly different behavior in retards to initial load, but the carousels between each should otherwise function similarly
